File: LeetcodeProblems Py/2678.py
class Solution:
    def countSeniors(self, details: List[str]) -> int:
        return sum(1 for detail in details if int(detail[11:13]) > 60)



class Solution:
    def countSeniors(self, details: List[str]) -> int:
        ans = 0
        temp = []
        for detail in details:
            if int(detail[11:13]) > 60:
                ans += 1
                temp.append(detail[11:13])
        print(temp)
        return ans

File: LeetcodeProblems Cpp/2678.cpp
class Solution {
public:
    int countSeniors(vector<string>& details) {
        int ans = 0;
        for (int i = 0; i < details.size(); i++) {
            std::string sub_str = details[i].substr(11, 2);
            if( std::stoi(sub_str) > 60) {
                ans++;
            }
        }
        return ans;
    }
};